src/atoms/typography/paragraph.tsx

```diff
@@ -0,0 +1,69 @@
+import React, { forwardRef } from "react";
+
+// -------------------------------------------------------------------------------------------------
+// #region Enums
+// -------------------------------------------------------------------------------------------------
+
+export enum ParagraphSizes {
+    XLarge = "-xlarge",
+    Large = "-large",
+    Base = "-base",
+    Small = "-small",
+    XSmall = "-xsmall",
+}
+
+// #endregion Enums
+
+// -------------------------------------------------------------------------------------------------
+// #region Interfaces
+// -------------------------------------------------------------------------------------------------
+
+export interface ParagraphProps {
+    children?: any;
+    cssClassName?: string;
+    dangerouslySetInnerHTML?: { __html: string };
+    id?: string;
+    ref?: React.RefObject<HTMLParagraphElement> | null;
+    size?: ParagraphSizes;
+}
+
+// #endregion Interfaces
+
+// -------------------------------------------------------------------------------------------------
+// #region Component
+// -------------------------------------------------------------------------------------------------
+
+const Paragraph: React.RefForwardingComponent<
+    HTMLParagraphElement,
+    ParagraphProps
+> = forwardRef(
+    (props: ParagraphProps, ref: React.Ref<HTMLParagraphElement>) => {
+        let cssClassNames: Array<any> = [];
+
+        if (props.cssClassName) {
```

Comment:
Prefer `StringUtils.hasValue` instead of nullsy checking

Reply:
Bah, missed this one